binancial/strategies/__init__.py

```
@@ -1 +1,2 @@
-from .mean_strategy import mean_strategy
\ No newline at end of file
+from .mean_reversion_crossover import mean_reversion_crossover
```

Ruff (0.8.2)

`.mean_reversion_crossover.mean_reversion_crossover` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)